### ISSUE TITLE
Add webforms libraries to the project composer.json

### ISSUE THREAD

**Issue by finnlewis** [localgovdrupal/localgov_microsites_project#47]
Our default install includes of LocalGov Microsites includes webform, but not the webform libraries.

This results in the error message on the site status page:

![image](https://github.com/localgovdrupal/localgov_microsites_project/assets/326588/e5b940c2-1b0d-4d1b-9d7c-b1108051e229)

We could set this up in our project template to save devs having to do that as part of the setup.

Here's the documentation on how to add the libraries with the the composer merge plugin.

https://www.drupal.org/docs/8/modules/webform/webform-frequently-asked-questions/how-to-use-composer-to-install-libraries-for-the-webform-module

Basically: 

```
composer require wikimedia/composer-merge-plugin
```

And add this to extras in composer.json:

```
        "merge-plugin": {
            "include": [
                "web/modules/contrib/webform/composer.libraries.json"
            ]
        }
```
